.github/workflows/docs.yml

```diff
@@ -0,0 +1,32 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      JULIA_PKG_SERVER: ''
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Set up Julia
+        uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.5
```

Comment:
We currently need to build the documentation on v1.5 because the output format of arrays has changed with v1.6.

Reply:
OK we can defer that fix to https://github.com/JuliaReach/SpaceExParser.jl/issues/50